packages/suite/src/actions/wallet/__tests__/discoveryActions.test.ts

```diff
@@ -156,6 +157,7 @@ jest.mock('@trezor/connect', () => {
             getAccountInfo,
             cancel: () => {},
         },
+        DEVICE,
```

Comment:
I find out that maybe better solution is to place `...jest.requireActual('@trezor/connect')` at very beginning of this object a then you don't have to care adding these piece by piece